core/src/test/java/hudson/PluginWrapperTest.java

```diff
@@ -48,13 +59,15 @@ public void optionalDependencyTest() {
 
     @Test
     public void jenkinsCoreTooOld() throws Exception {
+
         PluginWrapper pw = pluginWrapper("fake").requiredCoreVersion("3.0").buildLoaded();
         try {
             pw.resolvePluginDependencies();
             fail();
         } catch (IOException ex) {
             assertContains(ex, "Failed to load: fake (42)", "Jenkins (3.0) or higher required");
         }
+        Locale.setDefault(loc);
```

Comment:
Why here? `after()` will run anyways right?

Reply:
Yes, you are right. I forgot to remove it after moving it to the after().